reference/actions/error.md

```diff
@@ -8,20 +8,38 @@ Terminates the flow, sets its body template result as [`$error`](/reference/vari
 
 ## Syntax
 
-The `status` attribute defines the HTTP response status code. This is a
+The optional `status` attribute defines the HTTP response status code. This is a
 shortcut for a [`set-status` action](set-status.md). The default is `500`.
 
-The action body contains a JSON template reprensenting a JSON object.
+The action body contains a constant JSON string or a JSON template representing a JSON object.
+The object will be assigned to the `$error` variable.
+
+If the template is a string, it must be enclosed in double quotes (`"`), and its value will be
+assigned to the `$error/message` property.
+
+The following properties of the `$error` object are optional, but must have the specified 
+type if set and otherwise will receive the following default values:
```

Comment:
Das klingt für mich so, als käme der Default zum Tragen, wenn der Typ falsch ist.

"...type if set". If not set they will..."?